dali/server/daldap.cpp

```diff
@@ -226,6 +226,7 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
         if (!authenticated)
         {
             user->credentials().setPassword(password);
+            user->credentials().setSessionToken(udesc->querySessionToken());
```

Comment:
@RussWhitehead I think the suggested fix is that this is changed to

    if (!user->isSuperUser()) ....

with no authentication.  (I think it is equivalent to the changes in this PR.)